Utils.py

```diff
@@ -49,7 +49,6 @@ def as_simple_string(self) -> str:
 
 __version__ = "0.6.4"
 version_tuple = tuplize_version(__version__)
-version = Version(*version_tuple)
```

Comment:
setup.py imports version from Utils.

I am not sure which one we should keep. On one hand Version is a Named Tuple, so version_tuple is technically correct, but it might make sense to just call it version?

In any way, removing either one without some sort of phase out would be a breaking change, so this should maybe become
```py
version = version_tuple = tuplize_version(__version__)  # remove XXX right after next release
```
and a separate PR to actually remove one that gets merged as soon as the next version is out.

Reply:
I think find references let me down here originally. I find having the extra name here kinda weird, but changing its use in setup.py would probably belong in another PR. Aliasing it seems good to me.

Reply:
Also in terms of deciding if/when it should be removed, it's probably worth mentioning it was only added within the last couple of weeks in the manifest PR, so general breakage potential would probably be very low. It doesn't seem like it was mentioned in the PR or by any commentors, so I'm not sure if it was intended to possibly have a different value from `version_tuple` in the future, but I can't really imagine what purpose that would have given it was only used currently to set a default on min/max AP version.

Reply:
> probably best to just keep whichever is older, as it has less chance to break an apworld somewhere?

So we get rid of `version` and update `setup.py` accordingly.